Azure/azure-cli
# [SQL] `az sql midb restore`: Add support for cross-subscription restore

**Related command**


**Description**

Adding support for cross-subscription restore for managed database. One additional parameter - `--s` has been added in order to distinguish restore in same subscription with cross-subscription restore. 

**Testing Guide**

Automation couldn't be established as we need instance in both subscription, but only one of our subscriptions has pre-set infra for fast MI creation. Everything was manually tested and verified in Kusto and backend that correct params were passed.


---

This checklist is used to make sure that common guidelines for a pull request are followed.
